core/tests/protractor_utils/users.js

```diff
@@ -91,6 +91,7 @@ var createAndLoginAdminUser = function(email, username) {
 };
 
 exports.login = login;
+exports.completeSignup = _completeSignup;
```

Comment:
Why can't you use createAndLoginAdminUser() rather than exposing a private helper function?

Reply:
I tried using that but there is a problem with it in mobile devices. The `createAndLoginAdminUser` function calls the `login` function, the `_completeSignup` function. It then goes to the admin page, clicks on the `Roles` tab, searches for the update form, enters the username but when it tries to select role from the dropdown, the screen zooms in, shifts to the left and the "Update Role" button is not clickable post that. Therefore, I did not involve changing roles from the admin page.

Reply:
I think you should create a function named createAdminMobile() and createAndLoginAdminMobile() instead then? 
The completeSignup function should not be used out of context imo.

Reply:
Done, thanks!